src/alias/setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import re
+from setuptools import setup, find_packages
+
+# Inspired by https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-cdn/setup.py#L45
+extension_path = os.path.dirname(os.path.realpath(__file__))
+with open(os.path.join(extension_path, 'azext_alias/version.py'), 'r') as version_file:
```

Comment:
Do not hard code the `/` here. Use os.path.join for compatibility with both Linux and Windows.
Something like `os.path.join(extension_path, 'azext_alias', 'version.py')`